packages/react/src/component-helpers.tsx

```diff
@@ -26,7 +26,7 @@ export function RedlineBackground({height, hasBorder = true, ...rest}: PropsWith
         overflow: 'hidden',
         border: hasBorder ? '1px solid var(--base-color-scale-red-2)' : undefined,
         backgroundImage:
-          'linear-gradient(45deg, var(--base-color-scale-red-0) 12.5%, hsla(var(--base-color-scale-red-2-hsl) / 50%) 12.5%, hsla(var(--base-color-scale-red-2-hsl) / 50%) 50%, var(--base-color-scale-red-0) 50%, var(--base-color-scale-red-0) 62.5%, hsla(var(--base-color-scale-red-2-hsl) / 50%) 62.5%, hsla(var(--base-color-scale-red-2-hsl) / 50%) 100%)',
+          'linear-gradient(45deg, var(--base-color-scale-red-0) 12.5%, #ffb5b380 12.5%, #ffb5b380 50%, var(--base-color-scale-red-0) 50%, var(--base-color-scale-red-0) 62.5%, #ffb5b380 62.5%, #ffb5b380 100%)',
```

Comment:
These hex values are a fair bit harder to read IMO. I think we're losing readability and ease of correlation with base color scale tokens. 

Some ideas 💡
 
- As this is a JS file, how about we create a helper function that generates the final output, by using a named token as a parameter?
- Create new and locally scoped tokens with associated comments on what these hex values are and how they were created for future reference

Reply:
Since this is just for presentation and not product UI, I wonder if we should hold off for now and add some documentation specific tokens when we have access to the new build with alpha logic later? Otherwise we can just add them now and reference the hex, but I don't think that should be a blocker for this PR IMO.

Reply:
doc tokens sound good, yeah let's leave as is for now 👍